src/shared-components/dropdown/test.tsx

```diff
@@ -49,7 +49,13 @@ describe('<MobileDropdown />', () => {
   describe('UI snapshots', () => {
     it('renders correctly', () => {
       const tree = renderer
-        .create(<MobileDropdown options={options} />)
+        .create(
+          <MobileDropdown
+            onSelectChange={() => undefined}
+            borderRadius="4px"
```

Comment:
It's a little annoying to have to provide these for the Jest tests--however, I thought it was cleaner to add the optional argument w/ default at the parent component `Dropdown`, instead of keep the value optional there (without a default) and setting the default on both `MobileDropdown` and `DesktopDropdown`.